models/topics.model.js

```diff
@@ -13,4 +13,14 @@ exports.fetchEndpoints = () => {
   });
 };
 
-// endpoints.json
+exports.fetchArticleById = (id) => {
+  const paramAsNum = Number(id);
+  if (Number.isNaN(paramAsNum)) {
```

Comment:
Lets allow psql to pick up the error if it is going to, and only if it doesn't throw an error do we need to consider a custom error here.